### PR TITLE
Task/104 xslx config to json

### PR DESCRIPTION
**Pull request by mattiagiupponi**
With this PR we want to:
- Add missing library in README.md for validate schema example
- Adding configurations json validated by the script (tested)

The configuration are tested, still have some errors due some None value, but overall are working

### CODE REVIEW

File: app/scheduler/analysis_router.py [hunk 1]
```diff
@@ -9,9 +9,9 @@ class AnalysisRouter:
     db_key = 'analysis'
 
     def _check_name(self, name):
-        if name.lower() == "all_domains":
```
Comment (bieganowski) [inline app/scheduler/analysis_router.py:12]: I believe this is a fix from Christian - are you sure you want to merge this?

File: requirements.txt [hunk 2]
```diff
@@ -22,5 +22,4 @@ sympy==1.6.2
 pypika==0.43.0
 schema==0.7.2
 openpyxl==3.0.5
-django-postgres-copy==2.4.2
-openpyxl==3.0.5
```
Comment (bieganowski) [inline requirements.txt:26]: I believe `openpyxl` should be still present in the requirements - the Export Task uses it

**Review by bieganowski — CHANGES_REQUESTED** [review]

File: app/scheduler/analysis_router.py (hunk 1 above)
Comment (mattiagiupponi) [reply app/scheduler/analysis_router.py]: Rolledback, thanks for the comment 👍

File: requirements.txt (hunk 2 above)
Comment (mattiagiupponi) [reply requirements.txt:26]: Maybe from the pull-requst is not so visible, but the `openpyxl` was duplicated in the `requirements.txt`

**Review by bieganowski — APPROVED** [review]

**Pull request merged**